migrations/README

```diff
@@ -0,0 +1 @@
+Generic single-database configuration.
```

Comment:
what is this README?

Reply:
I guess this README is autogenerated when I migrate the sqlalchemy database, I didn't add it intentionally